cardano-node/src/Cardano/Node/Run.hs

```diff
@@ -117,7 +117,11 @@ runNode loggingLayer npm@NodeCLI{protocolFiles} = do
         Left err -> (putTextLn $ renderProtocolInstantiationError err) >> exitFailure
         Right (SomeConsensusProtocol p) -> pure $ SomeConsensusProtocol p
 
-    tracers <- mkTracers (ncTraceConfig nc) trace
+    txsProcessedCounter :: MVar Integer <- newMVar 0
+
+    tracers <- mkTracers (ncTraceConfig nc) trace txsProcessedCounter
+
+    txsProcessedThread <- Async.async $ traceTxsProcessed (appendName "metrics" trace) txsProcessedCounter 0
```

Comment:
is the name space: "cardano.node.metrics" ?

Reply:
Yes.